src/synapse/workload.py

```diff
@@ -167,6 +195,66 @@ def enable_metrics(container: ops.Container) -> None:
         raise EnableMetricsError(str(exc)) from exc
 
 
+def _get_mjolnir_config(access_token: str, room_id: str) -> typing.Dict:
+    """Create config as expected by mjolnir.
+
+    Args:
+        access_token: access token to be used by the mjolnir bot.
+        room_id: management room id monitored by the Mjolnir.
+
+    Returns:
+        Mjolnir configuration
+    """
+    default_content = f"""
+        dataPath: "/data/storage"
+        verboseLogging: false
+        logLevel: "INFO"
+        syncOnStartup: true
+        verifyPermissionsOnStartup: true
+        noop: false
+        fasterMembershipChecks: false
+        automaticallyRedactForReasons:
+        - "spam"
+        - "advertising"
+        protectAllJoinedRooms: false
+        backgroundDelayMS: 500
+        health:
+        healthz:
+            enabled: false
+            port: {MJOLNIR_HEALTH_PORT}
+            address: "0.0.0.0"
+            endpoint: "/healthz"
+            healthyStatus: 200
+            unhealthyStatus: 418
+        pollReports: false
+        displayReports: false
+        """
+    config = yaml.safe_load(default_content)
```

Comment:
Shouldn't we either use a dataclass that represent the configuration, or a template to avoid such "hardcoded" configuration ?

Reply:
Something like Jenkins [here](https://github.com/canonical/jenkins-k8s-operator/blob/main/src/jenkins.py#L254)?

Reply:
Changed :)